packages/network/src/logic/Node.ts

```diff
@@ -534,6 +542,19 @@ export class Node extends EventEmitter {
         }
     }
 
+    private startConnectionCleanUpInterval(interval: number): NodeJS.Timeout {
+        return setInterval(() => {
+            const peerIds = this.nodeToNode.getAllConnectionNodeIds()
+            const unusedConnections = peerIds.filter((peer) => !this.isNodePresent(peer))
+            if (unusedConnections.length > 0) {
+                this.logger.debug(`Disconnecting from ${unusedConnections.length} unused connections`)
+                unusedConnections.forEach((peerId) => {
+                    this.nodeToNode.disconnectFromNode(peerId, 'Unused connection')
```

Comment:
any harm in calling this while already disconnecting?

Reply:
Shouldn't be a problem:
https://github.com/streamr-dev/network-monorepo/blob/4ae0467d8472109de70a5e93523978df31a0b305/packages/network/src/connection/WebRtcConnection.ts#L179